thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -331,15 +331,17 @@ public Response createAnomalyFunction(@NotNull @QueryParam("dataset") String dat
       break;
     case HOURS:
     default:
-      windowDelayTime = 2;
+      windowDelayTime = 0;
       windowDelayTimeUnit = TimeUnit.HOURS;
     }
     anomalyFunctionSpec.setWindowDelayUnit(windowDelayTimeUnit);
     anomalyFunctionSpec.setWindowDelay(windowDelayTime);
 
     // setup detection frequency if it's minutely function
+    // the default frequency is determined by data granularity
+    // this frequency is a override to lower the system overload
     if (dataGranularity.getUnit().equals(TimeUnit.MINUTES)) {
-      TimeGranularity frequency = new TimeGranularity(30, TimeUnit.MINUTES);
```

Comment:
The previous setting is artificial limitation posted by old ingraph adaptor used. Which always have d15 minutes delay and we do not want to have the most recent one delayed and set its to 30 minutes to keep it safe.

Sitespeed data is in pinot, not from ingraph. Ingraph metric can be treated as real time for now.